src/sentry/api/endpoints/project_ownership.py

```diff
@@ -69,7 +69,7 @@ def validate(self, attrs):
             self.context["ownership"].project.organization,
         ):
             schema = create_schema_from_issue_owners(
-                attrs["raw"], self.context["ownership"].project_id, True
+                attrs["raw"], self.context["ownership"].project_id, add_owner_ids=True
```

Comment:
This and line 180 are small renaming fixes from the related ticket for [project_ownership](https://getsentry.atlassian.net/browse/WOR-2675)